README.rst

```diff
@@ -308,7 +308,7 @@ Recent Changes
   - Add Graph::capacity(), GraphMap::capacity()
   - Fix bug in Graph::reverse()
   - Graph and GraphMap have `quickcheck::Arbitrary` implementations,
-    if optional feature `quickcheck` is enabled.
+    if optional feature `check` is enabled.
```

Comment:
I'll have to think about this, this seems wrong. We'd like to keep the name of this feature flag.

Reply:
We want to fix our quickcheck code (inside src, not the tests) so that we use the randomness api that quickcheck is reexporting.

Reply:
I'm not familiar with quickcheck. So I think I' better leave this to you or anyone who's interested.